package netty;

option java_package = "com.ease.nogame.protobuf";
option java_outer_classname = "PBApp";

message MsgDesc {
    required string msgName = 1;
    optional bytes msgBytes = 2;
    optional int64 userId = 3;
    optional string token = 4;
    optional int32 errorCode = 5;
    optional string errorDesc = 6;
}

message PBHero {
        required int64 heroId = 1;
        required int64 dictId = 2;
        required int32 level = 3;
        required int32 exp = 4;
}

message C2SLogin {
        required string userName = 1;
        required string password = 2;
        required string deviceID = 3;
}

message S2CLogin {
        required int64 userId = 1;
        required string token = 2;
}

message C2SUserInfo {
        required int64 userId = 1;
}

message S2CUserInfo {
        required int64 userId = 1;
        required string userName = 2;
}

message C2SHeroInfo{
        
}

message S2CHeroInfo {
        repeated PBHero heroList = 1;
}

